powsybl-parent/pom.xml

```
@@ -429,7 +450,7 @@
                                             <message>Missing file "lombok.config" in project. Please create it starting with the following content "${powsybl.internal.lombok-import-line}".</message>
                                         </requireFilesExist>
                                         <evaluateBeanshell>
-                                            <condition>org.codehaus.plexus.util.FileUtils.fileRead(new File("${project.basedir}/lombok.config"), "UTF-8").startsWith("${powsybl.internal.lombok-import-line}")</condition>
+                                            <condition>org.codehaus.plexus.util.FileUtils.fileRead(new File("${escaped.project.basedir}/lombok.config"), "UTF-8").startsWith("${powsybl.internal.lombok-import-line}")</condition>
```

Would using java replace work then?
```suggestion
                                            <condition>org.codehaus.plexus.util.FileUtils.fileRead(new File("${project.basedir}/lombok.config".replaceAll("\\\\", "/")), "UTF-8").startsWith("${powsybl.internal.lombok-import-line}")</condition>
```

---

I tried to use java, but it fails before calling `replaceAll` since `${project.basedir}` is replaced by a string containing single `\`, which is not allowed in a (traditional) String.
But I haven't though of the [text blocks](https://docs.oracle.com/en/java/javase/15/text-blocks/index.html). I will test it.

---

With text blocks (using `"""` instead of `"`), it is not working either.

---

Oh I haven't understand it why Maven variable injection which cause the problem.  
Based on [this SO thread](https://stackoverflow.com/questions/63263586/maven-enforcer-how-to-access-maven-properties-from-beanshell-rule), I tested bellow suggestion and it seam to work now.  
Can you confirm?
```suggestion
                                            <condition>org.codehaus.plexus.util.FileUtils.fileRead(new File(URI.create("${project.baseUri}").resolve("lombok.config")), "UTF-8").startsWith("${powsybl.internal.lombok-import-line}")</condition>
```

---

The suggestion was applied instead of using a maven plugin execution.